hermes-api/src/main/java/pl/allegro/tech/hermes/api/endpoints/ConsumerEndpoint.java

```diff
@@ -0,0 +1,26 @@
+package pl.allegro.tech.hermes.api.endpoints;
```

Comment:
it should and we should kill this module all together. we only use it to build some proxy client in tests, so expanding it would be a mistake imo